components/eamxx/src/control/intensive_observation_period.cpp

```diff
@@ -191,6 +191,10 @@ initialize_iop_file(const util::TimeStamp& run_t0,
       EKAT_REQUIRE_MSG(field_rank <= 1,
                        "Error! Unexpected field rank "+std::to_string(field_rank)+" for iop file fields.\n");
       Field field(fid);
+      if (fl.has_tag(FieldTag::LevelMidPoint) or fl.has_tag(FieldTag::LevelInterface)) {
+        // Request packsize allocation for level layout
+        field.get_header().get_alloc_properties().request_allocation(Pack::n);
```

Comment:
It looks like you never really use `Pack`. The only time IOP uses packs, is with the "fake" `Pack1d`, just for the sake of being able to call the LinInterp interfaces (we should allow calling with a scalar view, btw).

If that's the case, then you should prob remove all the request (here and below) for a n-sized pack allocation, since IOP doesn't need it.

Reply:
We will use pack size n in the next PR.

And yes, scalar would be nice for lininterp. I'll add that to my list since I have a case here I can test.